Run Script - Vegas 13.cs

```diff
@@ -26,25 +32,40 @@ public class EntryPoint{
 			isPythonPresent = true;
 		}
 
-
-		Process p2 = System.Diagnostics.Process.Start(exeFilePath);
-		p2.WaitForExit(); // Start the bundled Python script and wait for it to finish.
-		lines = System.IO.File.ReadAllLines(Environment.ExpandEnvironmentVariables(@"%USERPROFILE%\AppData\Local\Temp\YTPMVE\errlog.txt"));
-		retry = bool.Parse(lines[2]);
-		if (p2.ExitCode != 0) {
-			if (isPythonPresent && retry) {
-				try{
-					System.Diagnostics.Process.Start("python", pyFilePath).WaitForExit(); // Start the Python script instead and wait for it to finish.
-				}
-				catch (System.Exception e){
-					MessageBox.Show("An error occurred while attempting to launch the script! \n\nError: " + e.Message + "\n\nDetails: \n" + lines[0], "Error" , MessageBoxButtons.OK, MessageBoxIcon.Error);
-					return;
+		/**
+		I know it's very clunky to have duplicated code like this, but there needs to be some sort of failsafe for if the executable is missing.
```

Comment:
the executable can't be missing since it's bundled on every commit, although adding a try catch would be nice it's not a good idea to add temporary code like this and push it to master, this would be more suitable for a dev branch

Reply:
Undid this change.